serverless/discover/about-serverless.adoc

```diff
@@ -6,15 +6,47 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-Serverless is a cloud computing model where application developers do not need to provision servers or manage scaling for their applications. These routine tasks are abstracted away by the platform, allowing developers to push code to production more quickly than in traditional models.
-
 {ServerlessProductName} simplifies the process of delivering code from development into production by reducing the need for infrastructure set up or back-end development by developers.
 
-Developers on {ServerlessProductName} can use the provided Kubernetes native APIs, as well as familiar languages and frameworks, to deploy applications and container workloads.
+Developers on {ServerlessProductName} can use the provided Kubernetes native APIs (Knative Serving and Eventing), as well as familiar languages and frameworks, to deploy applications and container workloads.
 
 {ServerlessProductName} is based on the open source link:https://knative.dev/docs/[Knative project], which provides portability and consistency for hybrid and multi-cloud environments by enabling an enterprise-grade serverless platform.
 
 [id="about-serverless-supported-configs"]
 == Supported configurations
 
 The set of supported features, configurations, and integrations for {ServerlessProductName}, current and past versions, are available at the link:https://access.redhat.com/articles/4912821[Supported Configurations page].
+
+[id="about-serverless-serving"]
+== Knative Serving
+
+Knative Serving on {product-title} enables developers to write link:https://www.redhat.com/en/topics/cloud-native-apps[cloud-native applications] using link:https://www.redhat.com/en/topics/cloud-native-apps/what-is-serverless[serverless architecture].
+
+Knative Serving supports deploying and managing cloud-native applications by providing a set of objects as Kubernetes custom resource definitions (CRDs) that define and control the behavior of serverless workloads on an {product-title} cluster.
+
+Developers use these CRDs to create custom resource (CR) instances that can be used as building blocks to address complex use cases. For example:
+
+* Rapidly deploying serverless containers.
+* Automatically scaling pods.
+
+[id="about-serverless-eventing"]
+== Knative Eventing
+
+Knative Eventing on {product-title} enables developers to use an link:https://www.redhat.com/en/topics/integration/what-is-event-driven-architecture[event-driven architecture] with serverless applications.
+
+An event-driven architecture is based on the concept of decoupled relationships between event producers that create events, and event  xref:../../serverless/knative_eventing/serverless-event-sinks.adoc#serverless-event-sinks[_sinks_], or consumers, that receive events.
```

Comment:
ISG recommends keeping sentences under 25 words. You have done a great job of explaining this, so feel free to disregard this comment if it doesn't help. What about breaking this up so that it really shines?

```suggestion
An event-driven architecture is based on the concept of decoupled relationships between event producers and event consumers. Event producers create events, and event  xref:../../serverless/knative_eventing/serverless-event-sinks.adoc#serverless-event-sinks[_sinks_], or consumers, receive events.
```

Reply:
love it, thank you!